docs/guides/migrating_to_active_mesh_transit_network.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: "aviatrix"
+page_title: "Migrating from Classic Aviatrix Encrypted Transit Network to Aviatrix ActiveMesh Transit Network"
+description: |-
+Aviatrix Transit Network Migration Guide
+---
+
+# Aviatrix Transit Network Migration Guide
+
+## USAGE
+For customers who are currently already using Terraform to manage their infrastructure including Classic Aviatrix Encrypted Transit Network prior to Controller 6.6 and are looking to migrate their Classic Aviatrix Encrypted Transit Network to Aviatrix ActiveMesh Transit Network and upgrade their Controller to 6.6+, please follow along for guidance on the migration process to ensure a smooth transition.
+
+---
+## Migration Steps
+
+If your Transit Network is built prior to Aviatrix software release 5.1, it’s very likely that the Transit Network is a non-ActiveMesh deployment where the IPSec tunnels between spoke gateways and transit gateways are in Active/Standby mode (i.e. only one IPSec tunnel is carrying the data traffic).
```

Comment:
I think you can add a sentence or two to reiterate why being non-ActiveMesh is an issue, because it seems unclear here. Mention that Non-ActiveMesh features/deployment model will be deprecated in 6.6 controller version and 2.21 provider version.

Maybe adding a relevant link to our official Aviatrix doc site saying "For more information regarding this deprecation, please see here" (here will have an embedded link)